chromium_src/chrome/browser/profiles/profile.h

```diff
@@ -0,0 +1,10 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "../../../../../chrome/browser/profiles/profile.h"
+
+// This is meant as a replacement for profile->GetProfileType() ==
+// GUEST_PROFILE, since GetProfileType is no longer public.
+bool IsGuestProfile(Profile* profile);
```

Comment:
`Profile::IsGuestSession`

Reply:
Profile::IsGuestSession is true for regular (not off-the-record) guest profile. This is meant to distinguish off-the-record profile of guest profile. GetProfileType (which is no longer public) would only return GUEST_PROFILE for off-the-record profile.